src/rpc/blockchain.cpp

```diff
@@ -938,8 +938,10 @@ static RPCHelpMan getblock()
     if (!request.params[1].isNull()) {
         if(request.params[1].isNum())
```

Comment:
might be easier to swap the if branches and make this branch bool?

Reply:
Yup you're right. Updated the PR according to your comments.